relaynet-jvm/src/main/java/tech/relaycorp/relaynet/CargoRelayServer.kt

```diff
@@ -0,0 +1,27 @@
+package tech.relaycorp.relaynet
+
+import kotlinx.coroutines.flow.Flow
+
+interface CargoRelayServer {
```

Comment:
My comments above about `CargoRelayClient` also apply here, but additionally: I think the core library shouldn't contain any server-specific API.